crmsh/sbd.py

```diff
@@ -199,7 +215,7 @@ def _determine_stonith_watchdog_timeout(self):
         res = SBDManager.get_sbd_value_from_config("SBD_WATCHDOG_TIMEOUT")
         if res:
             self._stonith_watchdog_timeout = -1
-        elif self._is_s390:
+        elif self._context.is_s390:
             self._stonith_watchdog_timeout = self.STONITH_WATCHDOG_TIMEOUT_DEFAULT_S390
```

Comment:
By default `stonith_watchdog_timeout` is 0, aka. disable. We only need to turn it on for diskless sbd. 
`-1` means paceamker will auto-calculate it based on SBD_WATCHDOG_TIMEOUT. 
`-1` is good enough. No need additional lines to explicitly deal with s390 since now profiles.yml concept deals with SBD_WATCHDOG_TIMEOUT flexibly.

I'm thinking to change
```
    def _determine_stonith_watchdog_timeout(self):
```

to
```
    def _enable_stonith_watchdog_timeout_for_diskless_sbd(self):
        if not self.diskless_sbd:
            return
        res = SBDManager.get_sbd_value_from_config("SBD_WATCHDOG_TIMEOUT")
        if res:
            self._stonith_watchdog_timeout = -1
```

Good to have @gao-yan 's double check here

Reply:
`stonith_watchdog_timeout=-1` relies on `SBD_WATCHDOG_TIMEOUT` to be explicitly set in sysconfig.  This is to handle the case where ` SBD_WATCHDOG_TIMEOUT` is unset in sysconfig.

But rather than handling it in here, it could be better to be initialized for s390 case also in above together with the normal case:
`self._stonith_watchdog_timeout = self.STONITH_WATCHDOG_TIMEOUT_DEFAULT`

Reply:
Yeah, to add the "else" clause does make sense,
```
        if res:
            self._stonith_watchdog_timeout = -1
        else:
            self._stonith_watchdog_timeout = self.STONITH_WATCHDOG_TIMEOUT_DEFAULT
```